instrumentation/src/com/intellij/rt/coverage/instrumentation/filters/ImplementerDefaultInterfaceMemberFilter.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2000-2020 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intellij.rt.coverage.instrumentation.filters;
+
+import com.intellij.rt.coverage.instrumentation.Instrumenter;
+import org.jetbrains.coverage.org.objectweb.asm.Label;
+import org.jetbrains.coverage.org.objectweb.asm.MethodVisitor;
+import org.jetbrains.coverage.org.objectweb.asm.Opcodes;
+
+/**
+ * Default interface member should be filtered out from implementer.
+ * Instructions list of such method consists exactly of:
+ * 0. LABEL
+ * 1. LINENUMBER
+ * 2. ALOAD 0
+ * 3. INVOKESTATIC to INTERFACE_NAME$DefaultImpls.INTERFACE_MEMBER
+ * 4. RETURN
+ * 5. LABEL
+ * A method is filtered out is it's instructions list matches this structure.
+ */
+public class ImplementerDefaultInterfaceMemberFilter extends MethodFilter {
```

Comment:
This is only about kotlin, maybe it is better the class name contain kotlin word?
Or put this into a kotin-something package.